socorro/unittest/lib/test_transform_rules.py

```diff
@@ -4,26 +4,15 @@
 
 import sys
 
-from nose.tools import eq_, ok_, assert_raises
-from mock import Mock, MagicMock, patch
-
 from configman.dotdict import DotDict
 from configman import Namespace
+from mock import Mock, MagicMock, patch
+import pytest
 
 from socorro.lib import transform_rules
 from socorro.unittest.testbase import TestCase
 
 
-def assert_expected(actual, expected):
-    assert actual == expected, "expected:\n%s\nbut got:\n%s" % (str(expected),
-                                                                str(actual))
-
-
-def assert_expected_same(actual, expected):
```

Comment:
curious. I'm surprised these were used in a limited scope and not removed or spread more widely

Reply:
Sometimes I feel like I'm digging through eras of Socorro conventions.